.github/workflows/check.yml

```diff
@@ -0,0 +1,27 @@
+name: Helm Chart CI
+
+on:
+  pull_request:
```

Comment:
**Optional** 🎨 Given how quickly this workflow runs, I suggest changing the event to `push` so that contributors are able to benefit from the lint check without opening pull requests and we perform a final check once changes are merged to `main`.

Reply:
This is a great idea:  providing that feedback loop can significantly reduce the overhead on reviewing PRs that are essentially optional config (eg; annotations) where the testing burden is simply confirming that the chart renders valid yaml.

Reply:
Interesting idea, I admittedly never considered CI runs outside of PRs. If you consider this valuable, I am happy to add it. 

If I understand correctly, we would no longer need to trigger on a PR then at all, since the this is run for every commit, and otherwise the PR can not be merged.

Reply:
The final check after merging might be required due to merging in outdated branches, right?